roles/dtc/common/templates/ndfc_route_control/ndfc_route_map.j2

```diff
@@ -0,0 +1,402 @@
+! This NDFC Route_Control_Route_maps config data structure is auto-generated
+! DO NOT EDIT MANUALLY
+!
```

Comment:
Same question here about separation into child include files for debugability / maintainability

Reply:
Done